Update taxonomy-api.md (#133)
amplitude/amplitude-dev-center: docs/analytics/apis/taxonomy-api.md

}
```

> Note that Taxonomy API requires your account to be on the **Enterprise plan** or have the **Govern add-on** enabled, so if your subscription does not qualify, the call will result in a `404 Bad Request`. 

Reviewer: ```suggestion
The Taxonomy API requires an **Enterprise plan**  account or the **Govern add-on** enabled. If your subscription doesn't qualify, the call results in a `404 Bad Request` response. 
```

@zheniasemenina Please avoid adding "Note" to comments.
